docker/Dockerfile

```diff
@@ -2,11 +2,13 @@ FROM paritytech/ci-linux:production as builder
 LABEL description="This is the build stage for Polkadot. Here we create the binary."
 
 ARG PROFILE=release
+ARG FEATURES
 WORKDIR /litentry-parachain
 
 COPY . /litentry-parachain
 
-RUN cargo build --$PROFILE
+# empty features (--features=) will be ignored
+RUN cargo build --$PROFILE --features=$FEATURES
```

Comment:
If  `$FEATURES` is empty, then the command reduces to `cargo build --$PROFILE`, right?

Reply:
> If `$FEATURES` is empty, then the command reduces to `cargo build --$PROFILE`, right?

It's actually `cargo build --release --features=`, which is the same as `cargo build --release`: as far as I tested.